umm-spec-lib/src/cmr/umm_spec/migration/version/granule.clj

```diff
@@ -54,6 +54,13 @@
     "application/xhdf5"
     mime-type))
 
+(defn- dissoc-track
+  "Migrate v1.5 Track to v1.4 by dissociating it from HorizontalSpatialDomain if applicable"
+  [g]
+  (if (get-in g [:SpatialExtent :HorizontalSpatialDomain :Track])
```

Comment:
Not sure if track 0 is a thing, but to be safe I'd wrap this check in `some?`.

Reply:
Track is a map. It will not be `0` or `false`.